llvm/lib/Transforms/Vectorize/VPlanVerifier.cpp

```diff
@@ -220,12 +219,29 @@ bool VPlanVerifier::verifyVPBasicBlock(const VPBasicBlock *VPBB) {
 
       for (const VPUser *U : V->users()) {
         auto *UI = cast<VPRecipeBase>(U);
-        // TODO: check dominance of incoming values for phis properly.
-        if (!UI ||
-            isa<VPHeaderPHIRecipe, VPWidenPHIRecipe, VPPredInstPHIRecipe,
-                VPIRPhi>(UI) ||
-            (isa<VPInstruction>(UI) &&
-             cast<VPInstruction>(UI)->getOpcode() == Instruction::PHI))
+        if (auto *Phi = dyn_cast<VPPhiAccessors>(UI)) {
+          for (unsigned Idx = 0; Idx != Phi->getNumIncoming(); ++Idx) {
+            VPValue *IncVPV = Phi->getIncomingValue(Idx);
```

Comment:
"Inc" is ambiguous - e.g., also stands for "Increment". Hence suggested renaming to `IncomingV` and `IncomingVPBB`.

Reply:
updated, thanks